src/systemverilog/systemverilog.test.ts

```diff
@@ -110,5 +110,44 @@ testTokenization('systemverilog', [
 			{ startIndex: 20, type: ''},
 			{ startIndex: 21, type: 'delimiter.curly.sv'}
 		]
-	}]
+	}],
+	// Strings
+	[{
+		line: 'var = "Hello world\n";',
```

Comment:
Can you also add string tests inside of functions? Take a look at the sample code spencer and i helped provide you can use those as reference

ex.
        $display ("display a message");
    bit [8*12:1] stringvar = "Hello world\n";
    parameter string default_name = "John Smith";
    $display( "%s : %d\n", c.name, c );